services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -174,6 +177,97 @@ class FlowHelper {
         } || existingFlows*.id.contains(newFlow.id)
     }
 
+    /**
+     * Check that all needed rules are created for a flow with protected path
+     *
+     * @param flowId
+     */
+    void verifyRulesOnProtectedFlow(String flowId) {
+        // TODO (andriidovhan) add possibility to use this method when we have more than 1 flow on a switch
+        // then update addFlow/deleteFlow/updateFlow
+        def mainFlowPath = northbound.getFlowPath(flowId).forwardPath
+        def srcMainNode = mainFlowPath[0]
+        def dstMainNode = mainFlowPath[-1]
+        def mainFlowTransitNodes = mainFlowPath[1..-2]
+        def protectedFlowPath = northbound.getFlowPath(flowId).protectedPath.forwardPath
```

Comment:
You already did this call on L188. Save to var instead of calling remote service twice